sample/src/main/res/layout/main_activity.xml

```diff
@@ -15,42 +15,56 @@
   -->
 
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
This is way more 2020.  I suck at UIs, so I kept with what I know well 😂 .

Reply:
Fair, I changed it because I needed to keep the button centered but have the loading indicator to the right of it, so it was this or RelativeLayout.